src/pages/Photos.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from "react";
+
+import '../index.css';
+import './Photos.css';
+
+import brendan from "../images/press/Brendan Shirtless, Drum Solo.jpg"
+import fullband from "../images/press/Full Band.jpg"
+import fullbandoutside from "../images/press/Full Band Outside.png"
+import connorscream from "../images/press/Connor Screaming.jpg"
+import connorlex from "../images/press/Connor and Lex.jpg"
+import scriptures from "../images/press/Scriptures Cover.png"
+import BasicPage from "../components/BasicPage";
+
+
+export const Photos = (props) => {
+
+  return (
+    <BasicPage>
+      <div class="photogallery">
+        <div className="photocol">
+          <img src={fullband} className="singlephoto"/>
```

Comment:
for every photo we use, we need to credit the photographer. if we don't know who, then we shouldn't use it.

Reply:
also this is super minor but the width of this page is just a smidge over the width of my monitor so it scrolls left and right just a lil bit. not a big deal but something i noticed